### ISSUE TITLE
Talk about DOIs leading to landing pages

### ISSUE THREAD

**Issue by augustfly** [augustfly/DOI4Archives#21]

**Comment by augustfly** [comment]
- [ ] find out what RDA means when he talks about the issue of "landing pages"
- [ ] identify different landing page examples
- [ ] discuss best practices for landing pages
- [ ] discuss embedded schema.org tags for landing pages (if by now this task list isn't so far removed from RDA's original idea)

**Comment by gilleslandais** [comment]
in Datacite , I often use the Pangea example (https://support.datacite.org/docs/landing-pages). 
Of dcourse, in the note  we can add a link to the Datacite Web page, we can also add an other example.

the Pangea example is interesting, it shows a view with expected information and  includes hidden information (for instance in schema.org) which can be harvested by machine: 

For instance the FAIRChecker tools assign a note to your landing page:
https://fair-checker.france-bioinformatique.fr/

**Comment by augustfly** [comment]
I agree that there are good examples of landing pages! But there's also a fair bit of variation out there (good or bad). I look forward to Raffaele's detailed concerns!

**Comment by BaptisteCecconi** [comment]
From my own experience on publishing datasets (see here for a list of the datasets published within the [MASER service repository](https://maser.lesia.obspm.fr/publications/doi/?lang=en), the Datacite landing-page recommendation should be followed. 

A landing-page should is the entry page for the digital object. For sanity checks, it should be clear to the reader that:
- what the digital object is about (title + summary)
- what is the DOI of the object (did I land where I should)

One should find the recommended citation (usually after the title and authors, as for a journal article publication).

One should also find a link to the data distribution(s) (in the [DCAT schema](https://www.w3.org/TR/vocab-dcat-3/#Class:Distribution) sense). Hence at least a web access to the dataset, or means to access it (e.g., through a VO service).

One should also find some description of the dataset: content, provenance, coverage, to help the reader's decision before access the data.

**Comment by BaptisteCecconi** [comment]
Here are examples of landing pages that are ok-ish, but not fully passing my criteria: 
- https://doi.org/10.17189/1519607 (NASA/PDS): there is a title, authors, DOI, summary... but there is no direct link to the data. The "Search/Access Data" link goes to a search interface, where the user should search again with some parameter (not specified, to be guessed)
- https://doi.org/10.48322/rgf7-3h67 (HPDE): there is everything about the citation, title, doi, summary, content... but the formatting is very close to the SPASE XML record. It's ok for user fluent with SPASE, not so much for outsiders. 
- https://doi.org/10.57780/esa-3xcjd4w (ESA): All required/recommended information is present, but the granuality (a single DOI for the full experience, not split with version ro processing levels...). And the data access goes to a search interface, where the user should search again for the products.

**Comment by augustfly** [comment]
I feel like we might be heading towards a table of extant landing pages and underlying issues! I have no concerns with this kind of analysis! 

It attacks a basic problem where I was only thinking about exploring something harder -- helping repositories embed schema.org tags on landing pages to help machines reach data directly. Though to be honest I was thinking about the generic solution rather than anything VO specific.

**Comment by raffaele-dabrusco** [comment]
Hi, I did not specifically refer to a particular issue, but wanted to convey the fact that some DOI issuing organizations would like to get astronomy-specific guidance on how landing pages should be. For the Chandra, we didn't have the resources to create from scratch landing pages and took advantage of our public interface to create landing pages for single entity DOIs, but if one is starting the effort from scratch, it would be useful to see recommendation on how landing pages should be created.
